.gitignore

```diff
@@ -1,3 +1,3 @@
-!/.gradle
-!/.idea
-!/gradle
+.gradle/
```

Comment:
The first thing I noticed when I opened up your PR was that it contained 792 files 😱 

I can see that you've added the folders containing most of these to your .gitignore file - which is absolutely the right thing to do - but now that they've been added to your git repository, they need to be manually removed before the rule sin .gitignore will come into effect.

[This stack overflow post](https://stackoverflow.com/questions/1139762/ignore-files-that-have-already-been-committed-to-a-git-repository) might be a good place to start

Reply:
I'd also look at adding the build and vendor folders here

Reply:
thank you, I paired with Elle just now to solve this